### PR TITLE
change links to relative ones

### PR DESCRIPTION
**Pull request by grego952**
**Description**

Changes proposed in this pull request:

- Change links to relative ones in the Keda documentation

**Related issue(s)**

Resolves https://github.com/kyma-project/keda-manager/issues/250

### CODE REVIEW

File: docs/user/_sidebar.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-- [Keda module](/docs/user/README.md)
-- [KEDA configuration](/docs/user/01-20-configuration.md)
-- [Keda module footprint](/docs/user/04-10-footprint.md)
-- [KEDA demo application](/docs/user/04-20-demo-applications.md)
-- [Keda CR Conditions](/docs/user/05-01-conditions.md)
+- [Keda module](README.md)
+- [KEDA configuration](01-20-configuration.md)
+- [Keda module footprint](04-10-footprint.md)
+- [KEDA demo application](4-20-demo-applications.md)
```
Comment (pbochynski) [inline docs/user/_sidebar.md:4]:
```suggestion
- [KEDA demo application](04-20-demo-applications.md)
```

File: docs/user/_sidebar.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
-- [Keda module](/docs/user/README.md)
-- [KEDA configuration](/docs/user/01-20-configuration.md)
-- [Keda module footprint](/docs/user/04-10-footprint.md)
-- [KEDA demo application](/docs/user/04-20-demo-applications.md)
-- [Keda CR Conditions](/docs/user/05-01-conditions.md)
+- [Keda module](README.md)
```
Comment (pbochynski) [inline docs/user/_sidebar.md:1]:
```suggestion
- [Home](/)
- [Keda module](README.md)
```

Comment (pbochynski) [reply docs/user/_sidebar.md:1]: Adding Home link will allow to get back to the main website when user enters Keda docs.
I also fixed a link to demo application (4-20-demo... -> 04-20-demo...)
You can test your branch with docsify to spot the problems:
- install docsify: `npm i docsify-cli -g`
- checkout kyma-project/kyma repository (git clone)
- start website locally from kyma root folder: `docsify serve docs`
- modify alias in the docs/index.html to point to your branch:
    ```
    '/keda-manager/(.*)': 'https://raw.githubusercontent.com/pbochynski/keda-manager/docsify/docs/$1'
    ```
    to
    ```
    '/keda-manager/(.*)': 'https://raw.githubusercontent.com/grego952/keda-manager/relative-links/docs/$1'
    ```

Comment (grego952) [reply docs/user/_sidebar.md:1]: Ok, thanks.

Comment (grego952) [reply docs/user/_sidebar.md:2]: Looks like everything works fine. 
Thank you @pbochynski for your support!

**Review by pbochynski — APPROVED** [review]

**Review by mmitoraj — APPROVED** [review]

**Pull request merged**